docrawl/docrawl_core.py

```diff
@@ -121,18 +103,22 @@ def _initialise_browser(self):
 
             if self.headless:
                 self.options.add_argument("--headless")
-
                 # For headless mode different width of window is needed
                 window_size_x = 1450
 
             try:
-                try:
-                    service = Service(GeckoDriverManager().install())
-                except Exception as e:
-                    service = None
-                    docrawl_logger.warning("GeckoDriverManager update was not successful - launching latest Firefox version instead"+str(e))
-                
-                self.browser = webdriver.Firefox(options=self.options, service=service, seleniumwire_options=sw_options)
+                service = Service(GeckoDriverManager().install())
```

Comment:
I was actually thinking of sticking to specific driver version, this might help to avoid some version incompatibility issues. Webdriver manager should have option for this.

Reply:
I agree.

Reply:
I disagree -> We want to keep the newest browser -> otherwise we will face incompatibilities with old browsers - why would we even introduce this feature?

Reply:
Dont forget in desktop version it needs to update user's browser